cmd/compose2exo/main.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/deref/exo/internal/import/compose"
+	"github.com/deref/exo/internal/external/compose"
```

Comment:
`/internal/external/` is kinda confusing :-P Alternative name suggestion: `compat`? Or maybe just move `compose` and `procfile` in to the `manifest` package as sub-packages?